src/js/formats/twoSided.js

```diff
@@ -0,0 +1,42 @@
+import twoSidedTemplate from '../text/twoSided.dot.html!text';
+import slider from '../lib/slider';
```

Comment:
There's a `helpers` directory in `formats` which contains a `buildCarousel` module. Perhaps `slider` belongs here too? Alternatively, I'd be equally happy to move the `buildCarousel` to `lib`, as long as we're consistent 😄